### DISCUSSION TITLE
Failing to download Tracker Data for "big" counts

### DISCUSSION THREAD

**Discussion by maddust** [opendatacam/opendatacam#534] [Q&A] [answered]
Hi ,

im getting a "Failed-network error" every time im  trying to download Tracker Data from High Volume Counts. im currently counting pedestrians and bikes from  12 to 24hr videos with around 98k tracked objects. 

I know this file can get very big ... but any advise how i can pull this data without failing? 

Currently im using 3.0.2 docker version - desktop .

also tried the api with no luck . only small counts can be downloaded.

**Comment by vsaw** [comment]
You could try connecting to the database using https://www.mongodb.com/products/compass

As for the error, could you attach the full error message? I'm curious to see why it fails.

**Reply by maddust** [reply]
thanks! i was able to download the tracker.json using mongodb compass it was 1.5gb 

https://imgur.com/6f7Ll9g


as for the error as soon as it pops out in the bottom of the browser i get Failed - Network error. 

https://imgur.com/8a2n3y4

log:


> Ready on http://localhost:8080
> Ready on http://172.18.0.3:8080
Success init db
(node:33) [DEP0001] DeprecationWarning: OutgoingMessage.flush is deprecated. Use flushHeaders instead.
First request on console stream
> Ready on http://localhost:8080
> Ready on http://172.18.0.3:8080
Success init db
(node:33) [DEP0001] DeprecationWarning: OutgoingMessage.flush is deprecated. Use flushHeaders instead.
First request on console stream

**Reply by vsaw** [reply]
I don't see any error on the logs or the screenshots. Could you provide a the complete logs, by any chance?

**Reply by maddust** [reply]
the second screenshot its what it shows in the bottom bar (double check)

log below:


> Ready on http://localhost:8080
> Ready on http://172.18.0.3:8080
Success init db
(node:33) [DEP0001] DeprecationWarning: OutgoingMessage.flush is deprecated. Use flushHeaders instead.
First request on console stream
New client, close previous stream
Process YOLO started
 CUDA-version: 10020 (11040), cuDNN: 7.6.5, GPU count: 1  
 OpenCV version: 4.1.1
 0 : compute_capability = 750, cudnn_half = 0, GPU: NVIDIA GeForce RTX 2070 
   layer   filters  size/strd(dil)      input                output
   0 conv     32       3 x 3/ 2    640 x 352 x   3 ->  320 x 176 x  32 0.097 BF
   1 conv     64       3 x 3/ 2    320 x 176 x  32 ->  160 x  88 x  64 0.519 BF
   2 conv     64       3 x 3/ 1    160 x  88 x  64 ->  160 x  88 x  64 1.038 BF
   3 route  2 		                       1/2 ->  160 x  88 x  32 
   4 conv     32       3 x 3/ 1    160 x  88 x  32 ->  160 x  88 x  32 0.260 BF
   5 conv     32       3 x 3/ 1    160 x  88 x  32 ->  160 x  88 x  32 0.260 BF
   6 route  5 4 	                           ->  160 x  88 x  64 
   7 conv     64       1 x 1/ 1    160 x  88 x  64 ->  160 x  88 x  64 0.115 BF
   8 route  2 7 	                           ->  160 x  88 x 128 
   9 max                2x 2/ 2    160 x  88 x 128 ->   80 x  44 x 128 0.002 BF
  10 conv    128       3 x 3/ 1     80 x  44 x 128 ->   80 x  44 x 128 1.038 BF
  11 route  10 		                       1/2 ->   80 x  44 x  64 
  12 conv     64       3 x 3/ 1     80 x  44 x  64 ->   80 x  44 x  64 0.260 BF
  13 conv     64       3 x 3/ 1     80 x  44 x  64 ->   80 x  44 x  64 0.260 BF
  14 route  13 12 	                           ->   80 x  44 x 128 
  15 conv    128       1 x 1/ 1     80 x  44 x 128 ->   80 x  44 x 128 0.115 BF
  16 route  10 15 	                           ->   80 x  44 x 256 
  17 max                2x 2/ 2     80 x  44 x 256 ->   40 x  22 x 256 0.001 BF
  18 conv    256       3 x 3/ 1     40 x  22 x 256 ->   40 x  22 x 256 1.038 BF
  19 route  18 		                       1/2 ->   40 x  22 x 128 
  20 conv    128       3 x 3/ 1     40 x  22 x 128 ->   40 x  22 x 128 0.260 BF
  21 conv    128       3 x 3/ 1     40 x  22 x 128 ->   40 x  22 x 128 0.260 BF
  22 route  21 20 	                           ->   40 x  22 x 256 
  23 conv    256       1 x 1/ 1     40 x  22 x 256 ->   40 x  22 x 256 0.115 BF
  24 route  18 23 	                           ->   40 x  22 x 512 
  25 max                2x 2/ 2     40 x  22 x 512 ->   20 x  11 x 512 0.000 BF
  26 conv    512       3 x 3/ 1     20 x  11 x 512 ->   20 x  11 x 512 1.038 BF
  27 conv    256       1 x 1/ 1     20 x  11 x 512 ->   20 x  11 x 256 0.058 BF
  28 conv    512       3 x 3/ 1     20 x  11 x 256 ->   20 x  11 x 512 0.519 BF
  29 conv     39       1 x 1/ 1     20 x  11 x 512 ->   20 x  11 x  39 0.009 BF
  30 yolo
[yolo] params: iou loss: ciou (4), iou_norm: 0.07, cls_norm: 1.00, scale_x_y: 1.05
  31 route  27 		                           ->   20 x  11 x 256 
  32 conv    128       1 x 1/ 1     20 x  11 x 256 ->   20 x  11 x 128 0.014 BF
  33 upsample                 2x    20 x  11 x 128 ->   40 x  22 x 128
  34 route  33 23 	                           ->   40 x  22 x 384 
  35 conv    256       3 x 3/ 1     40 x  22 x 384 ->   40 x  22 x 256 1.557 BF
  36 conv     39       1 x 1/ 1     40 x  22 x 256 ->   40 x  22 x  39 0.018 BF
  37 yolo
[yolo] params: iou loss: ciou (4), iou_norm: 0.07, cls_norm: 1.00, scale_x_y: 1.05
  38 route  35 		                           ->   40 x  22 x 256 
  39 conv     64       1 x 1/ 1     40 x  22 x 256 ->   40 x  22 x  64 0.029 BF
  40 upsample                 2x    40 x  22 x  64 ->   80 x  44 x  64
  41 route  40 15 	                           ->   80 x  44 x 192 
  42 conv    128       3 x 3/ 1     80 x  44 x 192 ->   80 x  44 x 128 1.557 BF
  43 conv     39       1 x 1/ 1     80 x  44 x 128 ->   80 x  44 x  39 0.035 BF
  44 yolo
[yolo] params: iou loss: ciou (4), iou_norm: 0.07, cls_norm: 1.00, scale_x_y: 1.05
Total BFLOPS 10.471 
avg_outputs = 372836 
 Allocate additional workspace_size = 26.22 MB 
Loading weights from traffic_v1_best.weights...Demo
net.optimized_memory = 0 
mini_batch = 1, batch = 8, time_steps = 1, train = 0 
nms_kind: greedynms (1), beta = 0.600000 
nms_kind: greedynms (1), beta = 0.600000 
nms_kind: greedynms (1), beta = 0.600000 
Done! Loaded 45 layers from weights-file 
Got first message from JSONStream
Error with message send by YOLO, not valid JSON
setvideoresolution

 seen 64, trained: 1625 K-images (25 Kilo-batches_64) 
video file: opendatacam_videos/2019-09-22_11_00.mp4
Video stream: 1280 x 720 
JSON_sender: new client 24
Restore counting areas
MJPG_sender: new client 26
MJPG_sender: new client 27
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
New client, close previous stream
New client, close previous stream
New client, close previous stream
New client, close previous stream
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON
Error with message send by YOLO, not valid JSON

**Comment by maddust** [comment, marked as answer]
also it looks like the tracker data has not getting deleted in mongodb when you delete the recordings from ODC. 

looking in compass my recordings folder is empty (when i delete recordings) 
but the tracker has still 17gb of data stored from previous counts
